package.json

```diff
@@ -9,6 +9,7 @@
     "typescript": "~5.5.0"
   },
   "scripts": {
+    "version": "changeset version && pnpm install --lockfile-only",
```

Comment:
We need to use custom version script because the default one does not update the lockfile by itself.